lizmap/modules/admin/controllers/logs.classic.php

```diff
@@ -228,13 +231,16 @@ public function emptyDetail()
         $rep = $this->getResponse('redirect');
 
         // Get counter
-        $cnx = jDb::getConnection('lizlog');
-
-        try {
-            $cnx->exec('DELETE FROM log_detail;');
-            jMessage::add(jLocale::get('admin~admin.logs.empty.ok', array('log_detail')));
-        } catch (Exception $e) {
-            jLog::log('Error while emptying table log_detail ');
+        if (\jAcl2::check('lizmap.admin.lizmap.log.delete')) {
```

Comment:
you could indicate this rights on the `$pluginParams` directly like the exemple for emptyCounter